chore: update readme (#24)
zeit-labs/zeitlabs-payments: README.rst

zeitlabs-payments
#################

A pluggable Django/Python plugin for Open edX that simplifies payment integration.

|pypi-badge| |ci-badge| |codecov-badge| |doc-badge| |pyversions-badge|
|license-badge| |status-badge|

Purpose
*******

The purpose of this repository is to provide a lightweight, Django/Python-based payment plugin for Open edX that simplifies how payments are handled in the platform.

> Existing options for payments in Open edX — like the legacy eCommerce service, heavy full-stack solutions such as WooCommerce or Magento, third-party integrations, or fragmented custom modules — are either outdated, overly complex, or misaligned with what Open edX really needs.

Reviewer: Please move comparision to a dedicated section:

```suggestion

### Comparision to Existing Solutions
There's more than one option for payment in Open edX. Most have their own flaws, and this is a quick comparision of why this is the most-viable solution:

```

@tehreem-sadat I've added a full table, please compare and review and put here as an RST table:

 - https://docs.google.com/spreadsheets/d/1WRknRx2SQZ7XN-L_NnUgdJ1BhyGsO-s_3NpIxuLFr1s/edit?gid=0#gid=0

> What Open edX requires is not a full eCommerce store, but reliable and straightforward payment flows — something simple enough for a learner to quickly purchase a course, yet flexible enough to scale for larger institutional deployments.

Reviewer: ```suggestion
What Open edX requires is not a full eCommerce store, but reliable and straightforward payment flows -- something simple enough for a the admin to make a course purchasable, yet flexible enough to scale for larger institutional deployments.
```

> In short, the plugin brings a simpler, smarter, and more maintainable approach to Open edX payments — one that works today and grows with tomorrow’s needs.

Reviewer: ```suggestion
This plugin brings a simpler, smarter, and more maintainable approach to Open edX payments -- one that works today and grows with tomorrow’s needs.
```

Features
********

- **Seamless Integration** – Built with Django/Python, feels native to Open edX.
- **Lightweight & Cost-Effective** – No extra servers or infrastructure overhead.
- **Plug-and-Play Setup** – Fast integration, easy to manage via the admin panel.
- **Familiar User Experience** – Checkout flow consistent with Open edX’s deprecated eCommerce.

> - **Future-Ready** – Starting with Payfort support, with more gateways, discounts, and coupon features on the roadmap.

Reviewer: ```suggestion
- **Future-Ready** – Starting with Payfort support, with more gateways, discounts, and coupon features on the roadmap.
- **Tailored for Open edX:** Supports course purchase, program purchase and subscriptions in addition to custom purchase models due to its native integration with the edx-platform.
- **No Burden** – No need to maintain another server, or have your team learn new technologies.
```
